extensions/vscode/src/views/homeView.ts

```diff
@@ -430,36 +433,49 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     });
   }
 
-  private getSelectionState(): HomeViewState {
-    const state = this.context.workspaceState.get<HomeViewState>(
+  private getSelectionState(): DeploymentSelector | null {
```

Comment:
Shouldn't this still be `HomeViewState` since that type is `DeploymentSelector | null` anyway?